### PR TITLE
Joseph Bravo / completed: string-manipulation-1

### PR DESCRIPTION
**Pull request by joseph-bravo**
image

### CODE REVIEW

File: string-manipulation-1/src/is-vowel.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* exported isVowel */
+/*
+create array of vowels capitalized. take arg and capitalize
+for every vowel in array, check if equal to arg. if true, then return true
+return false at the end when for loop finishes without returning true
+*/
+
+function isVowel(character) {
+  var vowels = ['A', 'E', 'I', 'O', 'U'];
+  for (var i in ['A', 'E', 'I', 'O', 'U']) {
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:10]: Why are you using a `for...in` loop on an array?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on the pseudocode!!

File: string-manipulation-1/src/is-vowel.js (hunk 1 above)
Comment (joseph-bravo) [reply string-manipulation-1/src/is-vowel.js]: even if order of array doesn't matter, should `for...in` be avoided for array looping?

**Review by codingcodymiller — APPROVED** [review]
Yes, you should stick to either `while` loops or `for` loops for arrays 🤓

**Pull request merged**